backend/plugin/db/mssql/mssql.go

```diff
@@ -66,6 +66,13 @@ func (d *Driver) Open(_ context.Context, _ storepb.Engine, config db.ConnectionC
 	// See: https://github.com/microsoft/go-mssqldb/issues/33
 	query.Add("tlsmin", "1.0")
 
+	// Add extra connection parameters if specified in the DataSource
```

Comment:
@h3n4l we should remove some of the defaults which could cause conflicts to extra params. Let's take a look together.

Reply:
Not related to this PR though.